### ISSUE TITLE
No stories names on Overall test results page

### ISSUE THREAD

**Issue by biatcha** [serenity-bdd/serenity-jbehave#177]
Hi, after upgrading to java 10, latest eclipse supported it(Oxygen.3a) and latest serenity dependency\plugin versions the issue with empty stories block starts for me. 
![z](https://user-images.githubusercontent.com/37931802/38138085-6e753740-3431-11e8-93f4-b2ea32745550.jpg)

 <properties>
  <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
  <serenity.version>1.9.4</serenity.version>
  <serenity.maven.version>1.9.4</serenity.maven.version>
  <serenity.jbehave.version>1.38.0</serenity.jbehave.version>
  <webdriver.driver>firefox</webdriver.driver>
  <webdriver.chrome.driver>chromedriver.exe</webdriver.chrome.driver>
  <serenity.take.screenshots>FOR_FAILURES</serenity.take.screenshots>
        <storyTimeoutInSecs>50000</storyTimeoutInSecs>
 </properties>

dependencies>
  <dependency>
   <groupId>net.serenity-bdd</groupId>
   <artifactId>serenity-core</artifactId>
   <version>${serenity.version}</version>
  </dependency>
  <dependency>
   <groupId>net.serenity-bdd</groupId>
   <artifactId>serenity-jbehave</artifactId>
   <version>${serenity.jbehave.version}</version>
  </dependency>
  <dependency>
   <groupId>org.slf4j</groupId>
   <artifactId>slf4j-simple</artifactId>
   <version>1.6.1</version>
  </dependency>
  <dependency>
   <groupId>net.serenity-bdd</groupId>
   <artifactId>serenity-junit</artifactId>
   <version>${serenity.maven.version}</version>
   <scope>test</scope>
  </dependency>
  <dependency>
   <groupId>org.codehaus.groovy</groupId>
   <artifactId>groovy-all</artifactId>
   <version>2.3.7</version>
  </dependency>
  <dependency>
   <groupId>junit</groupId>
   <artifactId>junit</artifactId>
   <version>4.12</version>

<plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.7.0</version>
                <configuration>
                    <source>1.7</source>
                    <target>1.7</target>
                </configuration>
            </plugin>

   <plugin>
    <groupId>org.apache.maven.plugins</groupId>
    <artifactId>maven-surefire-plugin</artifactId>
    <version>2.21.0</version>
    <configuration>
     <skip>false</skip>
    </configuration>
   </plugin>
   <plugin>
    <artifactId>maven-failsafe-plugin</artifactId>
    <version>2.21.0</version>
    <configuration>
     <includes>
       <include>**/*TestSuite.java</include>
     </includes>
     <reuseForks>true</reuseForks>
     <argLine>-Xmx1024m </argLine>
     <systemProperties>
      <webdriver.driver>${webdriver.driver}</webdriver.driver>
      <webdriver.chrome.driver>${webdriver.chrome.driver}</webdriver.chrome.driver>
      <phantomjs.binary.path>${phantomjs.binary.path}</phantomjs.binary.path>
      <webdriver.base.url>${webdriver.base.url}</webdriver.base.url>
      <serenity.take.screenshots>${serenity.take.screenshots}</serenity.take.screenshots>
      <metafilter>${metafilter}</metafilter>
                        <story.timeout.in.secs>${storyTimeoutInSecs}</story.timeout.in.secs>
      <serenity.restart.browser.frequency>${serenity.restart.browser.frequency}</serenity.restart.browser.frequency

**Comment by wakaleo** [comment]
Do you get any error messages?

**Comment by biatcha** [comment]
nope, any errors\warnings during tests execution and report generation. Just this empty block.  Pom attached as well, thanks!

**Comment by wakaleo** [comment]
Can you provide a sample project?

**Comment by biatcha** [comment]
do you mean project structure?
[reports generated.txt](https://github.com/serenity-bdd/serenity-jbehave/files/1863612/reports.generated.txt)

**Comment by wakaleo** [comment]
I mean a full project that reproduces the issue, ie which produces tags when’re  in JDK 8 and doesn’t in JDK 10

**Comment by biatcha** [comment]
not sure i can do it here due to nda , but can provide some in direct

**Comment by wakaleo** [comment]
You can email me a project, but it will need to work without any proprietary or local dependencies for me to be able to take a look.
